src/Analysis/Ast/Impl/Analyzer/PythonInterpreter.cs

```diff
@@ -60,6 +57,9 @@ public sealed class PythonInterpreter : IPythonInterpreter {
                     = UnknownType = new PythonType("Unknown", builtinModule, string.Empty, LocationInfo.Empty);
             }
             await _moduleResolution.LoadBuiltinTypesAsync(cancellationToken);
+
+            _stubResolution = new TypeshedResolution(sm);
```

Comment:
Was this intentional or a merge issue? (#723)

Reply:
As soon as typeshed init starts, it wants Unknown type. It least for me.

Reply:
@AlexanderSher may comment I guess. After merge all tests fail for me since init of typeshed yielded analysis and all tests failed. Lemme see what it wants...

Reply:
Perhaps my PR can be reverted if Alex's PR was the "true" fix. I know that when I tested before either of them, a module would be added and then make it back to the interpreter and crash trying to access _stubResolution.

Reply:
For me
```
>	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.PythonModule.Parse() Line 337	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.PythonModule.InitializeContent(string content) Line 220	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.PythonModule.PythonModule(Microsoft.Python.Analysis.Modules.ModuleCreationOptions creationOptions, Microsoft.Python.Core.IServiceContainer services) Line 113	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.PythonModule.PythonModule(string moduleName, string filePath, Microsoft.Python.Analysis.Modules.ModuleType moduleType, Microsoft.Python.Analysis.Types.IPythonModule stub, Microsoft.Python.Core.IServiceContainer services) Line 85	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.CompiledPythonModule.CompiledPythonModule(string moduleName, Microsoft.Python.Analysis.Modules.ModuleType moduleType, string filePath, Microsoft.Python.Analysis.Types.IPythonModule stub, Microsoft.Python.Core.IServiceContainer services) Line 32	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.BuiltinsPythonModule.BuiltinsPythonModule(string moduleName, string filePath, Microsoft.Python.Core.IServiceContainer services) Line 37	C#
 	Microsoft.Python.Analysis.dll!Microsoft.Python.Analysis.Modules.Resolution.MainModuleResolution.InitializeAsync(System.Threading.CancellationToken cancellationToken) Line 51	C#

```
so main resolver init starts parsing and then analyzing builtins. So it is unlucky and reaches analysis stage before we populate Unknown, it crashes.

Reply:
I'm fine with putting it back, I just hope that the other failure doesn't happen (if it was the scraper the whole time then moving it back up will be fine).

Reply:
No, old one was wrong too. Sorted it out.